benchmark/fs/bench-stat.js

```diff
@@ -0,0 +1,24 @@
+'use strict';
+
+const common = require('../common');
+const fs = require('fs');
+
+const bench = common.createBenchmark(main, {
+  n: [1e4],
+  kind: ['lstat', 'stat']
+});
+
+
+function main(conf) {
+  const n = conf.n >>> 0;
+  const fn = fs[conf.kind];
```

Comment:
I’ve simply replaced the only use of this variable with `fs[conf.kind]`. It doesn’t seem to make any difference, and I would expect any difference it makes to be pretty weak in comparison to the cost of the actual syscall/thread pool operation.